src/main/java/com/datatheorem/mobileappsecurity/jenkins/plugin/sendbuild/SendBuildAction.java

```diff
@@ -63,10 +47,10 @@
  * </p>
  */
 
-class SendBuildAction {
+public class SendBuildAction extends MasterToSlaveFileCallable<SendBuildMessage> {
 
     private final String apiKey;
-    private final PrintStream logger; // Jenkins logger
```

Comment:
logger is not serializable. I changed to the listener and then call getLogger() to send the log